packages/api/src/models/player.ts

```diff
@@ -117,15 +117,22 @@ export class PlayerModel {
     }
   }
 
+  public async getAllRegistered(): Promise<Array<Player>> {
+    // TODO: Remove mongoDB $exists from model
+    const vtos = await this.repository.get({
+      token: { $exists: true, $ne: undefined },
+    })
+
+    return vtos.map((vto) => new Player(vto))
+  }
+
   public async getMany(paginationParams: {
     limit: number
     offset: number
   }): Promise<Array<Player>> {
     // TODO: Remove mongoDB $exists from model
     const vtos = await this.repository.getSortedBy(
-      {
-        token: { $exists: true },
-      },
+      {},
```

Comment:
```suggestion
      token: { $exists: true, $ne: undefined },
```

Reply:
This change breaks the following tests:

  ● Route /leaderboard › should return leaderboard values for each entity › players
  ● Route /leaderboard › should return leaderboard values for each entity AFTER TRADE › players
  ● Route /leaderboard › should return correct values when PAGINATION params are given

Reply:
Should I fix the tests or revert this change?

Reply:
The tests fixed themselves after some rebase